bindings/rust/lib.rs

```diff
@@ -0,0 +1,52 @@
+//! This crate provides quick_check language support for the [tree-sitter][] parsing library.
+//!
+//! Typically, you will use the [language][language func] function to add this language to a
+//! tree-sitter [Parser][], and then use the parser to parse some code:
+//!
+//! ```
+//! let code = "";
+//! let mut parser = tree_sitter::Parser::new();
+//! parser.set_language(tree_sitter_quick_check::language()).expect("Error loading quick_check grammar");
+//! let tree = parser.parse(code, None).unwrap();
+//! ```
+//!
+//! [Language]: https://docs.rs/tree-sitter/*/tree_sitter/struct.Language.html
+//! [language func]: fn.language.html
+//! [Parser]: https://docs.rs/tree-sitter/*/tree_sitter/struct.Parser.html
+//! [tree-sitter]: https://tree-sitter.github.io/
+
+use tree_sitter::Language;
+
+extern "C" {
+    fn tree_sitter_quick_check() -> Language;
```

Comment:
`s/quick_check/pascal/g` :thinking:

Reply:
Thanks for stopping by, much appreciated!

I did notice that the version numbers were a bit weird, but I cross-checked with some official Tree-Sitter grammars and they [looked the same](https://github.com/tree-sitter/tree-sitter-python/blob/b466ffb4d7ce0a833754429ad8d3a02e950ed74c/Makefile#L1), so I thought it was supposed to be this way. No idea how the "quick_check" got in there.

I hope I don't have to manually update a gazillion files every time I bump the version number. Guess I'll have to familiarize myself with GitHub workflows.

Reply:
> I hope I don't have to manually update a gazillion files every time I bump the version number.

Unfortunately you do, until the CLI gets a command for it. But it doesn't matter if you don't publish packages.